packages/ott-vis-panel/src/components/CorePanel.tsx

```diff
@@ -29,6 +30,15 @@ const getStyles = () => {
 			left: 0;
 			padding: 10px;
 		`,
+		legendBox: css`
+			position: absolute;
+			bottom: 0;
+			right: 0;
+			padding: 10px;
+			background-color: #ffffff;
+			border: 1px solid #000000;
+			border-radius: 5px;
+		`,
```

Comment:
Don't use emotion css (iirc this was in a previous review and also in person)

Reply:
yes you mentioned it prior, i put it back in to see if it would render and forgot to get rid of it.